### ISSUE TITLE
Dynamically loading kernel module and device tree

### ISSUE THREAD

**Issue by herry13** [fruit-testbed/fruit-agent#4]
Some kernel modules and device trees must be loaded to activate Raspberry Pi features such as SPI and I2C. The loading can be done dynamically simply by changing the configuration on the server or local, and then deploy it on a running system using `modprobe` and `dtoverlay` as discussed in https://github.com/fruit-testbed/node-management/issues/22.

**Comment by herry13** [comment]
081d568f99d3cad7f5117e2d96761d46139f5dc8 implements this feature with an enhancement introduced in b5a197bdbca7f50d9b2b6aebf8191ad8bcde8507.

**Comment by herry13** [comment]
To load a kernel module, just add it to field `modules` in `fruit.json`. For example,

```
{
  "modules": "i2c-dev",
  ...
}
```

Multiple modules can be loaded simply by defining them in the same field separated by space.

**Comment by herry13** [comment]
To load device-tree, just add it to field `dtparam` in `fruit.json`. For example:

```
{
  "dtparam": "spi=on i2c_arm=on",
}
```

**Comment by herry13** [comment]
implemented in 081d568f99d3cad7f5117e2d96761d46139f5dc8
